decidim-accountability/app/views/decidim/accountability/import_projects_mailer/import.html.erb

```diff
@@ -0,0 +1 @@
+<p><%= t(".success") %></p>
```

Comment:
Can we add also some information about the import here in another paragraph, such as:
```
3 results were imported from projects.
```

Note the difference in the string between singular and plural.